clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/loadbalancing/GoogleLoadBalancedBackend.groovy

```diff
@@ -39,17 +39,6 @@ class GoogleLoadBalancedBackend {
    */
   String serverGroupUrl
 
-  static BalancingMode toBalancingMode(String mode) {
```

Comment:
Can't you now drop the constant `Strings` up above and do comparisons directly against the `enum` values?